src/Model/Contact.php

```diff
@@ -2757,12 +2757,14 @@ public static function addRelationship(array $importer, array $contact, array $d
 		return null;
 	}
 
-	public static function removeFollower($importer, $contact)
+	public static function removeFollower(array $contact)
 	{
-		if (($contact['rel'] == self::FRIEND) || ($contact['rel'] == self::SHARING)) {
+		if (!empty($contact['rel']) && (($contact['rel'] == self::FRIEND) || ($contact['rel'] == self::SHARING))) {
```

Comment:
What about `if (in_array($contact['rel'] ?? [], [self::FRIEND, self::SHARING])) {` instead?

Reply:
I support this.